### ISSUE TITLE
 ISV Debugger Setup, Step 1 of 7 fails

### ISSUE THREAD

**Issue by jvg123** [forcedotcom/salesforcedx-vscode#5032]
### Summary

The "SFDX: Create and Set Up Project for ISV Debugging." command fails at step 1 with error:
```
Starting SFDX: ISV Debugger Setup, Step 1 of 7: Creating project

10:30:57.908 sfdx project generate --name <project> --output-dir <path> --template standard
 ›   Warning: project generate is not a sf command.
Did you mean project generate? [y/n]: 
 ›   Error: Run sf help for a list of available commands.
10:31:08.778 sfdx project generate --name <project> --output-dir <path> --template standard
 ended with exit code 127
```
It is not possible to interact with the prompt to enter `y`

Running the `sfdx project generate` command manually works fine

### Steps To Reproduce:

Follow steps 1 - 9 from https://developer.salesforce.com/tools/vscode/en/apex/isv-debugger

Original project had a dash in the name. But running without that also failed in the same way

### Expected result

ISV Debugger project created

### Actual result

Error:
```
Starting SFDX: ISV Debugger Setup, Step 1 of 7: Creating project

10:30:57.908 sfdx project generate --name <project> --output-dir <path> --template standard
 ›   Warning: project generate is not a sf command.
Did you mean project generate? [y/n]: 
 ›   Error: Run sf help for a list of available commands.
10:31:08.778 sfdx project generate --name <project> --output-dir <path> --template standard
 ended with exit code 127
```

### Additional information

N/A

**Salesforce Extension Version in VS Code**: v58.9.1

**SFDX CLI Version**: @salesforce/cli/2.3.8 darwin-x64 node-v16.9.1

**OS and version**: macOS Monterey version 12.6.7 (21G651

**Comment by CristiCanizales** [comment]
Hello @jvg123! This is a known issue and we're currently working on a fix. We will update when we have it and release it. Sorry for the inconveniences!

**Comment by RitamAgrawal** [comment]
@jvg123 We have released v58.11.0. Will you please try running `SFDX: Create and Set Up Project for ISV Debugging` with the latest version and let us know?
